### DISCUSSION TITLE
Support for Statamic 4

### DISCUSSION THREAD

**Discussion by itsmesaskovic** [spatie/statamic-responsive-images#222] [Ideas]
We love this Statamic addon! 

Now we do have a small challenge. Statamic 4 just got released and upgrading our website is blocked due to the conflict in version support.

`Problem 1
    - Root composer.json requires spatie/statamic-responsive-images ^3.1 -> satisfiable by spatie/statamic-responsive-images[v3.1.0, v3.1.1, v3.1.2, v3.1.3].
    - spatie/statamic-responsive-images[v3.1.0, ..., v3.1.3] require statamic/cms ~3.3 -> found statamic/cms[v3.3.0-beta.1, ..., 3.4.x-dev] but it conflicts with your root composer.json require (^4.0).`

Are there plans for upgrading to support Statamic v4?

**Comment by riasvdv** [comment]
Check out #221
